CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ master (unreleased)
 
 * Support objects created with Object.create(null). fixes [#468](https://github.com/mozilla/nunjucks/issues/468)
 
+* Support ESNext iterators, using Array.from. Merge of
+  [#760](https://github.com/mozilla/nunjucks/pull/760)
```

Comment:
Did you mean #1058? :sweat_smile:

Reply:
Whoops, but I also want to mention that PR and I didn't understand what the format was. Would you mind making a PR that corrects this, please?

Reply:
#1061

Reply:
Thank you.